bin/lib/common.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs')
+const extend = require('extend')
+const { cyan, bold } = require('colorette')
+const util = require('util')
+
+module.exports = {}
+module.exports.loadConfig = loadConfig
+
+async function loadConfig (program, options) {
+  let argv = extend({}, options, { version: program.version() })
+  let configFile = argv['configFile'] || './config.json'
+
+  try {
+    const file = await util.promisify(fs.readFile)(configFile)
+
+    // Use flags with priority over config file
+    const config = JSON.parse(file)
+    Object.keys(config).forEach((option) => {
+      argv[option] = argv[option] || config[option]
+    })
+  } catch (err) {
+    // No file exists, not a problem
+    console.log(cyan(bold('TIP')), 'create a config.json: `$ solid init`')
```

Comment:
Think about using a logger library (likely a new issue). My favourite is [loglevel](https://www.npmjs.com/package/loglevel) (because it's simple and supports plugins).

[winston](https://www.npmjs.com/package/winston) is popular also.

This way you can fine-tune the logging.

Reply:
Definitely a new issue, the required changes would be server-wide. 
That said, it /should/ eventually be done, since one of the most frequently requested features about logging is timestamps, which our current logger does not support.

Reply:
Logging is great for long-running processes (and the server has logging), but this is just a short one-time job.

Reply:
Ah, actually, I should probably reuse `debug` that's laid out via `lib/debug.js`.

Reply:
We might consider a logger anyway, but yeah, that should be its own issue.

Reply:
https://github.com/solid/node-solid-server/issues/920